src/sanaprotocolbuilder/urls.py

```diff
@@ -6,7 +6,9 @@
 from webapp import views as webappViews
 from rest_framework.authtoken import views as restViews
 
-urlpatterns = patterns('',
+urlpatterns = patterns(
+    '',
```

Comment:
Nevermind, that's supposed to be there. It's for prefixing the views (https://docs.djangoproject.com/en/1.7/topics/http/urls/#the-view-prefix)